tests/beaker/lib/lvm_helper.rb

```diff
@@ -78,41 +96,47 @@ def is_correct_format?(agent, volume_group, logical_volume, format_type)
 # ==== Examples
 #
 # remove_all(agent, '/dev/sdb', 'VolumeGroup_1234', 'LogicalVolume_fa13')
-def remove_all(agent, pv=nil, vg=nil, lv=nil)
-  step 'remove logical volume if any:'
-  if lv
-    if lv.kind_of?(Array)
-      lv.each do |logical_volume|
-        on(agent, "umount /dev/#{vg}/#{logical_volume}", :acceptable_exit_codes => [0,1])
-        on(agent, "lvremove /dev/#{vg}/#{logical_volume} --force")
+def remove_all(agent, pv=nil, vg=nil, lv=nil, aix=nil)
```

Comment:
You should have the "aix" parameter be a boolean flag instead of a string parameter. So the default would be ```aix=false```.